ui/raidboss/data/04-sb/raid/o3s.js

```diff
@@ -297,8 +319,10 @@
     },
     {
       id: 'O3S The Queen\'s Waltz: Crystal Square',
-      regex: / 14:230A:Halicarnassus starts using/,
-      regexDe: / 14:230A:Halikarnassos starts using/,
+      regex: / 14:230A:Halicarnassus starts using The Queen'S Waltz/,
```

Comment:
Should the S be capitalized here in `The Queen's Waltz`?

Reply:
Don't think so. Made it small like the other 3 The Queen's Waltz